packages/eslint-plugin-boxel/docs/rules/missing-card-api-import.md

```diff
@@ -0,0 +1,100 @@
+# Disallow usage of card-api with missing imports with auto-fix (`@cardstack/boxel/missing-card-api-import`)
+
+💼 This rule is enabled in the ✅ `recommended` config.
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+<!-- end auto-generated rule header -->
+
+Auto-fixes missing imports for card-api related symbols in your code.
+
+If you refer to a card-api symbol without importing it:
```

Comment:
"symbol" is misleading. we do have a bunch of symbols that we export from the api, but that is not what you mean here. I think "export"  or "named export" is a better name here